src/nbtool/src/nbtool/util/NBConstants.java

```diff
@@ -64,8 +66,9 @@ private STATUS(int i) {
 	private static Map<String, Class<? extends ViewParent>[]> setupPossible() {
 		Map<String, Class<? extends ViewParent>[]> map = new HashMap<String, Class<? extends ViewParent>[]>();
 		
-		map.put("YUVImage", new Class[]{ImageView.class, ZoomImageView.class, CrossBright.class});
+		map.put("YUVImage", new Class[]{ImageView.class, ZoomImageView.class, CrossBright.class, EdgeView.class});
 		map.put("STATS", new Class[]{BotStatsView.class});
+		map.put("prot-RobotLocation", new Class[]{FieldView.class});
```

Comment:
Pretty sure that's what I meant, I don't have a computer on me right now. I will get back to you asap.

Sent from my iPhone

On Jun 10, 2015, at 22:58, philipkoch <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/nbtool/src/nbtool/util/NBConstants.java<https://github.com/northern-bites/nbites/pull/716#discussion_r32186471>:

>               map.put("STATS", new Class[]{BotStatsView.class});
> +             map.put("prot-RobotLocation", new Class[]{FieldView.class});


Do you mean 'proto-' to use ProtoBufView?

—
Reply to this email directly or view it on GitHub<https://github.com/northern-bites/nbites/pull/716/files#r32186471>.